lib/java/ydoc-server/src/main/java/org/enso/ydoc/Main.java

```diff
@@ -14,6 +14,8 @@ public class Main {
   private Main() {}
 
   public static void main(String[] args) throws Exception {
+    System.setProperty("helidon.serialFilter.pattern", "javax.management.**;java.lang.**;java.rmi.**;javax.security.auth.Subject;!*");
```

Comment:
I found this [advice on stackoverflow](https://stackoverflow.com/questions/76952056/how-to-make-jconsole-connect-to-a-helidon-jvm) and it does seem to help. I can connect to the process with JVisualVM **polyglot sampler** now.

![polyglot sampler](https://github.com/enso-org/enso/assets/26887752/38cbe3ae-a28b-4bb2-b6a8-91d86047eeee)

I'd say the first edit is fine on my computer.